lapce-app/src/terminal/view.rs

```diff
@@ -291,7 +294,12 @@ impl View for TerminalView {
             }
 
             if term_bg != bg {
-                let rect = Size::new(char_width, line_height)
+                let term_fg = *config.get_color(LapceColor::TERMINAL_FOREGROUND);
+                if check_contrast(&fg, &bg) && term_fg == fg {
+                    fg = term_bg;
+                };
+                let width = char_width * 1.1;
```

Comment:
I'm not sure if this is the right way to do this, but without this modifier the background only renders until the end of the cell. This looks like there is a frame around the cell.